app/tests/test_helper.js

```diff
@@ -0,0 +1,19 @@
+import jsdom from 'jsdom';
```

Comment:
At server-side we call the dir with tests `test`: https://github.com/shri-2015-org/shrimp/tree/master/server/test
Even though I like `tests` better, we should stick to `test` for consistency, or rename in both places.